setup.py

```diff
@@ -18,7 +18,7 @@
             "pytest-cov[toml]",
             "sqlalchemy~=1.3.0",
             "sqlalchemy-utils>=0.32,<0.36.4",
-            "indexd @ git+ssh://git@github.com/NCI-GDC/indexd.git@2.11.1-rc.3#egg=indexd",
+            "indexd",
         ],
         "pytest_indexd": [
```

Comment:
What's the fundamental difference between "dev" and "pytest_indexd"?
I wonder if they can be merged into one?
(new ticket, new PR)

Reply:
No, they can not. 
pytest_indexd is a test plugin to delegate database setup to pytest-postgresql and should be able to help us parallel our testing.